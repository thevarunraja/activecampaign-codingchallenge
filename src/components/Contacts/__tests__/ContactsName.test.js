import "@testing-library/jest-dom/extend-expect";
import { cleanup, render } from "@testing-library/react";
import React from "react";
import ContactsName from "../ContactName";

afterEach(cleanup);

describe("ContactsName Component", () => {
  it("matches snapshot", async () => {
    const { container } = render(
      <ContactsName firstName={"Barbara"} lastName={"Doe"} />
    );
    expect(container).toMatchSnapshot();
  });
});
