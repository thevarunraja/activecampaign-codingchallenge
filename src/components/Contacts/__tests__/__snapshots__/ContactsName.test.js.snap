// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactsName Component matches snapshot 1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="avatar"
    >
      B
      B
    </div>
    <div
      class="name"
    >
      <a
        class="a"
        href="#"
      >
        Barbara
         
        Barbara
      </a>
    </div>
  </div>
</div>
`;
